### PR TITLE
Circle generator and a small PRNG from Bob Jenkins

### PR DESCRIPTION
**Pull request by kkinion**

### CODE REVIEW

File: lib/generators/circles.rb [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    num.each_with_index do |n, idx|
+      val += (n.to_i) << 8 * idx
+    end
+
+    [ val ].pack('L>')
+  end
+
+end
+
+RubyGenerator.run if __FILE__ == $0
```
Comment (kedji) [inline lib/generators/circles.rb]: Wrong generator

File: lib/generators/circles.rb [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  end
+
+  def next_chunk2
+    amp = (@r.rand(1) + 1) * 128
+    vals = [
+      Math::PI * @r.rand,
+      Math::PI * @r.rand,
+    ]
+    num = [
+      Math.sin(-vals[0]) * amp + amp + 128 - amp,
```
Comment (kedji) [inline lib/generators/circles.rb]: What's with the `+ amp - amp`? Won't that just cancel out?

File: lib/generators/circles.rb [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+    val = 0
+    num.each_with_index do |n, idx|
+      val += (n.to_i) << 8 * idx
+    end
+
+    [ val ].pack('L>')
+  end
+
+  def next_chunk2
```
Comment (kedji) [inline lib/generators/circles.rb]: What is `next_chunk2`? If it's just a reference function and isn't designed to be called, can you put that in a comment?

Comment (kkinion) [inline lib/generators/circles.rb]: Oops - I'd meant to delete that entire function. It'll be removed in the next commit.

File: lib/generators/circles.rb (hunk 1 above)
Comment (kkinion) [inline lib/generators/circles.rb]: Will fix this in the next commit.

**Pull request merged**